_commands/basics/date.md

```diff
@@ -3,22 +3,58 @@
 
 date
 -------
-TODO: Add documentation for this command by submitting a pull request.
 <!-- one line explanation would go here -->
-
+`date` returns a system time the user specified.
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ date
+Tue Feb  6 12:42:38 EST 2018
+$ date +%Y%m%d
+20180206
 ~~~
 
 <!--more-->
 
+This command returns the standard current time. With different options and parameters, it can also return other system time with different formats.
+
 ### Useful Options / Examples
 
+~~~ bash
+$ date --help
+~~~
+Run this command to display a list of options.
+
+The useful options include:
+
+~~~ bash
+%a  show abbreviated weekday name (eg. Mon)
+%b  show abbreviated month name (eg. Jan)
+%c  show date and time
+%s  show the current seconds from 1970-01-01 00:00:00 UTC
+~~~
+
+For more help, run
+~~~ bash
+man date
+~~~
+to view the manual of this command.
+
 #### Example command
 
+~~~ bash
+$ date +%s
+1517939576
+~~~
+
 ##### Break it down
 
 #### Example command
```

Comment:
These example commands can link in with my above comment about the options (so combine the two sections). This would make it clearer for a user to see how to use these commands.